### DISCUSSION TITLE
I am not able to fetch the changes done in profile

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#848] [Q&A] [answered]
I changed the name in profile from admin dashboard but still the profile.name field is empty. Why is it so?
and after changing the profile name from client side also it reflects in admin dashboard but the response which I get remains empty.
`client.records.update('profiles', profileId, data)`

**Comment by ganigeorgiev** [comment, marked as answer]
> I changed the name in profile from admin dashboard but still the profile.name field is empty. 

Which `profile.name` field? If you mean `client.authStore.model.profile.name`, then yes, currently you have to manually call `client.users.refresh()` to refresh the stored user model state.

**Reply by ganigeorgiev** [reply]
Also, make sure that you are using the latest [PocketBase v0.7.9](https://github.com/pocketbase/pocketbase/releases/tag/v0.7.9) since previously there was a regression in the tables-listing reactivity that may be related to your issue.

**Reply by rajesh6161** [reply]
> > I changed the name in profile from admin dashboard but still the profile.name field is empty.
> 
> Which `profile.name` field? If you mean `client.authStore.model.profile.name`, then yes, currently you have to manually call `client.users.refresh()` to refresh the stored user model state.

thanks this is what I was looking for
